src/dashboards/loader.ts

```diff
@@ -13,13 +13,13 @@ async function findSyntheticMonitoringFolder(): Promise<FolderInfo> {
   const backendSrv = getBackendSrv();
   const folders = (await backendSrv.get(`api/folders`)) as FolderInfo[];
   for (const folder of folders) {
-    if (folder.title === 'SyntheticMonitoring') {
+    if (folder.title === 'Synthetic Monitoring') {
```

Comment:
any strong feelings about *not* renaming the existing (if any) folder?

Reply:
IMO with space looks better :thinking: